CHANGES.rst

```diff
@@ -663,7 +663,7 @@ Bug fixes
 
 - ``astropy.constants``
 
-  - The constants ``Ry`` and ``u`` are now properly used inside the 
+  - The constants ``Ry`` and ``u`` are now properly used inside the
```

Comment:
Can you please update this to remove the extraneous whitespace change in this commit?  It makes merging difficult.